project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -106,20 +109,20 @@ export default class HeatMap extends React.Component {
     const {latitudeNorth, latitudeSouth, longitudeWest, longitudeEast } = tileFromTileId(tileid)
     const bounds = [[latitudeNorth, longitudeWest], [latitudeSouth, longitudeEast]];
 
-    this.refs.map.leafletElement.fitBounds(bounds);
+    this.refs.map.leafletElement.fitBounds(this.getMapBounds(bounds));
   }
 
   formatLeafletBounds(bbox) {
```

Comment:
This method is now redundant with `getMapBounds` so it can be inlined (i.e., remove `formatLeafletBounds` in favor of `getMapBounds`).